src/main/java/com/training/springpractice/errors/BadRequestException.java

```diff
@@ -0,0 +1,11 @@
+package com.training.springpractice.errors;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class BadRequestException extends RuntimeException {
```

Comment:
Please, take a look at **@RestControllerAdvice**, maybe that will help you to centralize your errors management.